plugins/dependencies/src/main/resources/refreshVersions-rules/jetbrains-version-alias-rules.txt

```diff
@@ -0,0 +1,2 @@
+org.jetbrains.exposed:exposed-*
+    ^^^^^^^^^.^^^^^^^
```

Comment:
Can you add the rules as parameter to DependencyGroup?
https://github.com/jmfayard/refreshVersions/blob/56fdfbb452cc7928f4fb136b67640f1f0f3ad370/plugins/dependencies/src/main/kotlin/dependencies/Kodein.kt#L17-L23


Same for your other pull requests

Reply:
We have a duplicated rule in this PR now @jmfayard

I think we should either keep it like it was for dependencies in the JetBrains group, or move them all, but I don't think we really want that, and we definitely don't want that in this PR.